pkg/retry/retry.go

```diff
@@ -9,6 +9,10 @@ const (
 	Interval = 15 * time.Second
 	// Timeout is the total time we will wait for an event to occur.
 	Timeout = time.Minute * 10
+	// TCPConnTimeout is the maximum amount of time for a single TCP connection to establish
+	TCPConnTimeout = time.Second * 45
```

Comment:
How did you decide on 45 seconds?

Reply:
A 45 second timeout gives 15 seconds of downtime before the next dial attempt. Consistent with our [interval between API calls](https://github.com/openshift/windows-machine-config-operator/blob/master/pkg/retry/retry.go#L8-L9)